### DISCUSSION TITLE
How to use veth pair interface for test?

### DISCUSSION THREAD

**Discussion by dingrui37** [troglobit/smcroute#188] [Q&A]
I add a pair of veth(vethHR<--->vethRH) and set vethRH to another netns，then use smcroutectl to add multicast route, but it does work， why?
```
ubuntu@OptiPlex-3000:~$ sudo smcroutectl add vethHR 239.0.10.1 vethH1
smcroutectl: ipc: mroute: inbound vethHR is not a known phyint
```

**Comment by troglobit** [comment]
Works for me, I use veth pairs in the SMCRoute test suite.  Maybe you could share with us how you set it all up?  I assume you've set IP addresses on all interfaces correctly?
